background/init.js

```diff
@@ -166,14 +166,17 @@ startMooltipass = function () {
                 // Deal both with RAW DATA and FORM DATA
                 // If we notice the extension being slow because of it intercepting too much data, uncomment the following line
                 //if (details && details.type === "xmlhttprequest") { // Raw data (multipart posts, etc)
+                if (details && details.type != "csp_report") {     
```

Comment:
what's csp_report?

Reply:
It's auto generated requests when CSP policy violated(and CSP used by NoScript and uBlock to block JS)